https://issues.redhat.com/browse/ACM-6946 Observability doc updates (#5444)
open-cluster-management/rhacm-docs: observability/observability_enable.adoc

[#managing-observability-service]
= Managing the observability service

> When you enable the observability component, you can use metrics and configure dashboards to make cluster resource information visible, help you save cost, and prevent unnecessary events.

Author: ```suggestion
When you enable the observability component, you can use metrics and configure Grafana dashboards to make cluster resource information visible, help you save cost, and prevent unnecessary events.
```

+
All the pods in `open-cluster-management-observability` namespace for Thanos, Grafana and Alertmanager are created. All the managed clusters connected to the {product-title-short} hub cluster are enabled to send metrics back to the {product-title-short} Observability service.

. Validate that the observability service is enabled and the data is populated by launching the Grafana dashboards. Click the *Grafana link* that is near the console header, from either the console _Overview_ page or the _Clusters_ page.
+
You can also access the {ocp-short} 3.11 Grafana dashboards with the following URL: `https://$ACM_URL/grafana/dashboards`. Select the folder named _OCP 3.11_ to view the {ocp-short} 3.11 dashboards.
+
*Note:* If you want to exclude specific managed clusters from collecting the observability data, add the following cluster label to your clusters: `observability: disabled`.

> . Verify that the `multicluster-observability-operator` pod is being deployed by the `multiclusterhub-operator` deployment by viewing the `labels` section of the `multicluster-observability-operator` deployment. Run the following command to view the deployment:

Author: ```suggestion
. Verify that the `multicluster-observability-operator` pod is being deployed by the `multiclusterhub-operator` deployment by viewing the `labels` section of the `multicluster-observability-operator` deployment. Run the following command to get the `multicluster-observability-operator` deployment and display the labels associated with the resource:
```

+
[source,bash]
----

> need command here

Author: @dislbenn will get command for this

Reviewer: This command will get the `multicluster-observability-operator` deployment  and display the labels associated with the resource:
```bash
$ oc get deploy multicluster-observability-operator -n open-cluster-management --show-labels

NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE   LABELS
multicluster-observability-operator   1/1     1            1           35m   installer.name=multiclusterhub,installer.namespace=open-cluster-management

```

> need command here

Author: Adding suggested command

```suggestion
oc get deploy multicluster-observability-operator -n open-cluster-management --show-labels

NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE   LABELS
multicluster-observability-operator   1/1     1            1           35m   installer.name=multiclusterhub,installer.namespace=open-cluster-management
```

+
All the pods in `open-cluster-management-observability` namespace for Thanos, Grafana and Alertmanager are created. All the managed clusters connected to the {product-title-short} hub cluster are enabled to send metrics back to the {product-title-short} Observability service.

> . Validate that the observability service is enabled and the data is populated by launching the Grafana dashboards. Click the *Grafana link* that is near the console header, from either the console _Overview_ page or the _Clusters_ page.

Reviewer 2: Since there are two separate actions, I would make them two separate steps:

1. Validate that the observaiblity service is enabled and the data is populated by launching the Grafana dashboards. 
2. Click the *Grafana link* that is near the console header, from either the console _Overview_ page or the _Clusters_ page.

> . Validate that the observability service is enabled and the data is populated by launching the Grafana dashboards. Click the *Grafana link* that is near the console header, from either the console _Overview_ page or the _Clusters_ page.

Author: Incorporating suggestion after peer review
```suggestion
. Validate that the observability service is enabled and the data is populated by launching the Grafana dashboards. 

. Click the *Grafana link* that is near the console header, from either the console _Overview_ page or the _Clusters_ page.
```

+

> You can also access the {ocp-short} 3.11 Grafana dashboards with the following URL: `https://$ACM_URL/grafana/dashboards`. Select the folder named _OCP 3.11_ to view the {ocp-short} 3.11 dashboards.

Reviewer 2: Since this is part of the procedure, I'd make it a step. Again, split it up into multiple steps, since there are multiple actions. 

So the whole thing would be: 

.Procedure

1. Validate that the observaiblity service is enabled and the data is populated by launching the Grafana dashboards.
2. Click the Grafana link that is near the console header, from either the console Overview page or the Clusters page.
3. Access the {ocp-short} 3.11 Grafana dashboards with the following URL: `https://$ACM_URL/grafana/dashboards`. 
4. To view the {ocp-short} dashboards, select the folder named _OCP 3.11_ . 
5. To verify that the `multicluster-observability-operator` pod is being deployed by the `multiclusterhub-operator` deployment, view the `labels` section of the `multicluster-observability-operator` deployment.
6. To get the `multicluster-observability-operator` deployment and display the labels associated with the resource, run the following command: 

< provide command > 

7. *Optional:* If you want to exclude specific managed clusters from collecting the observability data, add the following cluster label to your clusters: `observability: disabled`.

If I am understanding correctly, the note is more of an optional step, and I think it'd make a better flow if it were given after the command.

+
*Note:* If you want to exclude specific managed clusters from collecting the observability data, add the following cluster label to your clusters: `observability: disabled`.

> . Verify that the `multicluster-observability-operator` pod is being deployed by the `multiclusterhub-operator` deployment by viewing the `labels` section of the `multicluster-observability-operator` deployment. Run the following command to get the `multicluster-observability-operator` deployment and display the labels associated with the resource:

Reviewer 2: See my above comment for all the suggested revisions for the procedure steps.

. Validate that the observability service is enabled and the data is populated by launching the Grafana dashboards. 

. Click the *Grafana link* that is near the console header, from either the console _Overview_ page or the _Clusters_ page.
+
.. Alternatively, access the {ocp-short} 3.11 Grafana dashboards with the following URL: `https://$ACM_URL/grafana/dashboards`.
+
.. To view the {ocp-short} 3.11 dashboards, select the folder named _OCP 3.11_ .
 
. Access the `multicluster-observability-operator` deployment to verify that the `multicluster-observability-operator` pod is being deployed by the `multiclusterhub-operator` deployment. Run the following command:
+
[source,bash]
----
oc get deploy multicluster-observability-operator -n open-cluster-management --show-labels

NAME                                  READY   UP-TO-DATE   AVAILABLE   AGE   LABELS
multicluster-observability-operator   1/1     1            1           35m   installer.name=multiclusterhub,installer.namespace=open-cluster-management

> ----

Author: ```suggestion
----

```
